src/routes/news/components/__tests__/Tab.spec.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react'
+import { shallow } from 'enzyme'
+import Tab from '../Tab'
+
+describe('Tab', () => {
+  const TU_NEWS = 'tu'
+  const type = TU_NEWS
+  const active = true
+  const destination = '/testcity/en/news/local'
+  const t = (key: ?string): string => key || ''
+
+  it('should render and match snapshot', () => {
+    expect(shallow(
+      <Tab
+        type={type}
+        active={active}
+        destination={destination}
+        t={t}
+      />
+    )).toMatchSnapshot()
```

Comment:
It would be very good if this tests would assert fro actual behavior of the component. Like that it displays some items.